packages/graphql/lib/get-graphql-error.js

```diff
@@ -0,0 +1,36 @@
+const getHeaders = (status, headers) => {
+  if (status === 429) {
+    const retryAfter = headers.get('retry-after');
+    return retryAfter ? { 'retry-after': retryAfter } : {};
+  }
+  return {};
+};
+
+module.exports = error => {
+  if (!error.networkError || !error.networkError.response) {
+    return error;
+  }
+
+  const { stack, response } = error.networkError;
+  const { status: statusCode, statusText, headers: responseHeaders } = response;
+  const status = statusCode === 429 ? 429 : 500;
+  const headers = getHeaders(status, responseHeaders);
+  const message = statusCode === 200 ? 'Not Acceptable' : statusText;
+  const fetchError = Object.assign(new Error(message), {
+    name: 'HopsGraphQlError',
```

Comment:
`HopsFetchDataError`?

Reply:
Depends on how we want to handle it for `development` envs later on. Since ppl might be using Redux and GraphQL in parallel, it could be helpful to know the source exactly rather than to have a generic fetch error.